src/Monolog/Processor/IntrospectionProcessor.php

```diff
@@ -55,7 +55,7 @@ public function __invoke(array $record)
             return $record;
         }
 
-        $trace = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS);
+        $trace = debug_backtrace((version_compare(PHP_VERSION, '5.3.6') < 0) ? 2 : DEBUG_BACKTRACE_IGNORE_ARGS);
```

Comment:
I'm ok with using the (int) PHP_VERSION_ID.

`(PHP_VERSION_ID >= 50306) ? DEBUG_BACKTRACE_IGNORE_ARGS : 2`